src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;

```

Comment:
The `ReturnOperation` class should be placed in the `strategy` package as per the checklist requirement to gather classes with the same purpose/common logic in a corresponding package.